classes/admin/admin_setting_userselection.php

```diff
@@ -41,7 +41,12 @@ public function __construct($name, $visiblename, $description, $defaultsetting =
      * @return array The ids of the currently selected users.
      */
     public function get_setting() {
-        return explode(',', $this->config_read($this->name));
+        $setting = $this->config_read($this->name);
+        if (!$setting) {
+            return [];
+        }
+
+        return explode(',', $this->config_read($setting));
```

Comment:
This isn't right and returns an error when the setting has a value. I think this line should be;
`return explode(',', $setting);`

Reply:
Good catch!!

Reply:
Just gave that a force push.